controller_interface/CMakeLists.txt

```diff
@@ -6,7 +6,6 @@ find_package(catkin REQUIRED COMPONENTS roscpp hardware_interface pluginlib)
 
 # Declare catkin package
 catkin_package(
-  DEPENDS boost
```

Comment:
It should be `Boost` in the `CMakeLists.txt` and `<depend>boost</depend>` in `package.xml` (assuming `format="2"`, but it's similar for format 1).